features/mbedtls/importer/Makefile

```diff
@@ -35,9 +35,19 @@ TARGET_PREFIX_CRYPTO:=../mbed-crypto/
 TARGET_SRC:=$(TARGET_PREFIX)src
 TARGET_INC:=$(TARGET_PREFIX)inc
 TARGET_TESTS:=$(TARGET_PREFIX)TESTS
+
+# New folder structure is introduced here for targets with Secured-Partition-Environment
+# and Non-Secured-Partition-Environment, below documentation for each folder:
+# COMPONENT_PSA_SRV_IMPL - include secure service business logic implementation
+# code. For example mbedCrytpo or secure time core logic
```

Comment:
Not a blocker - there are a few typos here, and the phrasing can be better. You don't need to say "New folder". In a year's time, the comment will still be there, and it won't be new! Revising the language I'd suggest:

# The folder structure is for targets that compile to the Secured-Partition-Environment
 # and Non-Secured-Partition-Environments. Documentation for each folder:
 # COMPONENT_PSA_SRV_IMPL - includes secure service business logic implementation
 # code. For example Mbed Crypto or the secure time core logic